package.json

```diff
@@ -1,23 +1,138 @@
 {
-  "name": "distribution-not-selected",
-  "description": "Choose a distribution by running 'npm run apply:copay' or 'npm run apply:bitpay'.",
-  "primary-package-json": "See the tempate in app-template/package.json",
-  "pre-commit": "unstage-package",
-  "changes": "changes to this file can be commited with the --no-verify option",
-  "scripts": {
-    "postinstall": "npm run apply:copay && echo && echo \"Repo configured for standard Copay distribution. To switch to the BitPay distribution, run 'npm run apply:bitpay'.\" && echo",
-    "start": "echo && echo \"Choose a distribution by running 'npm run apply:copay' or 'npm run apply:bitpay'.\" && echo",
-    "apply:copay": "npm i fs-extra@0.30 && cd app-template && node apply.js copay && cd .. && npm i",
-    "apply:bitpay": "npm i fs-extra@0.30 && cd app-template && node apply.js bitpay && cd .. && npm i",
-    "unstage-package": "git reset package.json",
-    "clean-all": "git clean -dfx"
+  "name": "NavPay",
+  "description": "A Secure NavCoin Wallet",
```

Comment:
I don't think we're really supposed to commit changes to this file.
But techinically that's only because of a hangover from Copay's need to build copay and bitpay from a single repo.

Reply:
Yeah we aren't. but that was because of how copay did it. Since we just have the one app, it makes a lot more sense to do it this way.